default.nix

```diff
@@ -24,7 +24,13 @@
       '';
 
       build-wasm-app = self.callPackage ./build-wasm-app.nix {};
+
+      # Issue happens when combining pkgsStatic & custom cross stdenv.
+      # We need to force a normal busybox here to avoid hitting a
+      # weird bootstrapping issue.
+      busybox-sandbox-shell = super.busybox-sandbox-shell.override { busybox = self.busybox; };
     })] ++ overlays;
+    config = { allowUnsupportedSystem = true; };
```

Comment:
`allowUnsupportedSystem` doesn't seem right....

Reply:
The alternative is to add wasm32-unknown to systems/doubles.nix here: https://github.com/NixOS/nixpkgs/blob/master/lib/systems/doubles.nix. I'd like to avoid it though since wasm32-unknown-unknown is not actively maintained (https://github.com/jfbastien/musl has been archived).